yarn.lock

```diff
@@ -5361,7 +5361,7 @@ diff@^5.1.0:
   resolved "https://registry.yarnpkg.com/diff/-/diff-5.1.0.tgz#bc52d298c5ea8df9194800224445ed43ffc87e40"
   integrity sha512-D+mk+qE8VC/PAUrlAU34N+VfXev0ghe5ywmpqrawphmVZc1bEfn56uo9qpyGp1p4xpzOHkSW4ztBd6L7Xx4ACw==
 
-diffhtml@1.0.0-beta.30:
+diffhtml@^1.0.0-beta.30:
```

Comment:
Huh! How come it stayed in the lockfile? Is it still in one of our `package.json` files? Or does something else depend on it?

Reply:
I think it just needs a `yarn` run 😃

Reply:
ah, no 😄 it's in `app/assets/javascripts/discourse/package.json` now 😅